etc/docs/build.ts

```diff
@@ -2,13 +2,12 @@
 
 import { parse, stringify } from '@iarna/toml';
 import { exec as execCb } from 'child_process';
-import { readFileSync, writeFile as writeFileCb } from 'fs';
+import { writeFile, readFile } from 'fs/promises';
```

Comment:
since the file was renamed, we should also update the README that talks about generate-docs.ts